### ISSUE TITLE
question/feature: forcing a dhcp option

### ISSUE THREAD

**Issue by tomaskrcka** [coova/coova-chilli#389]
Few months ago I needed a dhcp option 43 for android. It's letting know that connection is expensive.
Current implementation doesn't support forcing a dhcp option, just if it's requested by client.
What do you think about this possibility and implementation of this feature ?
It can be also useful for example when sending options to PXELinux. 
I have some dirty patch I used for it, so I could adapt it.

**Comment by gbaligh** [comment]
Hi,

Is this not covered by "dhcpopt" - "Set a DHCP option using a hex string" ?

**Comment by tomaskrcka** [comment]
Hi, it's sent just if requested by client.

**Comment by gbaligh** [comment]
Hi,

If we send it regardless of client request, can this cause a problem ?

**Comment by xOneca** [comment]
The client will simply ignore the options not supported.

**Comment by gbaligh** [comment]
This is exactly what I meant by my question 😉 ...

**Comment by tomaskrcka** [comment]
Yeah, that's correct. The client will ignore it. I've tested it with many clients, also with android phones and no any problem. And specially in my case, option that the connection is too expensive, it was working properly.

**Comment by wstoettinger** [comment]
Force-sending options is supported by many DHCP servers. i would very much appreciate this! 
@tomaskrcka can you provide your dirty patch here until this is implemented?
